src/lib/components/product_variants/ProductVariants.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Row from '../row/Row';
+import ProductHelper from '../../utils/ProductHelper';
+
+const CLASS_ROOT = 'tp-product-variants';
+
+export default class MoreCanvases extends Component {
+  render() {
+    this.productHelper = new ProductHelper();
+
+    const variants = this.props.variants.map((variant, i) => {
+      const imageUrl = variant._embedded.defaultSku.images[0].url;
```

Comment:
Maybe on next pass, but instead of grabbing the first image you might want to grab the image with the type you're looking for. Perhaps type = 'preview' (I'd have to look at the data again).

Reply:
Yeah good call

Reply:
Oh, it gets tricky, because not all products have a "preview" or "mockup" image (Laptop cases and mugs)

But we can default those to a different image other than mockup for now.

Reply:
Current quick fix for those two variants

```
mockupImage(variant) {
    const defaultSku = variant._embedded.defaultSku;
    var image = defaultSku.images.filter(image => {
      return (
        image.type === 'mockup' ||
        (defaultSku.productType === 'laptop-case' && image.type === 'front') ||
        (defaultSku.productType === 'mug' && image.type === 'front')
      );
    });
    return image[0];
  }
```
https://github.com/BustedTees/teepublic-react/pull/2/files#diff-2c98df148c4244cf3b1f40711a91ffb2R23